frontend/src/components/Volunteer/volunteer.jsx

```diff
@@ -8,195 +8,67 @@ class Volunteer extends Component {
-                        onChange={this.onChange}
-                      >
-                        <span className="buttonSpan">Submit</span>
-                      </button>
-                    </p>
-                  </div>
-                  <br />
-                </Col>
-                <Col className="volunteerMain" xs={12} sm={12} md={12} lg={5}>
-                  <h2 className="mb-4">Volunteer!</h2>
-                  <p className="query">
-                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
-                    Ipsa molestias at laudantium, distinctio nulla quo minima
-                    delectus consequuntur aut incidunt ipsam quidem repellat
-                    fugiat consequatur nihil odit quam voluptas earum.Lorem
-                    ipsum dolor sit amet consectetur adipisicing elit. Ipsa
-                    molestias at laudantium, distinctio nulla quo minima
-                    delectus consequuntur aut incidunt ipsam quidem repellat
-                    fugiat consequatur nihil odit quam voluptas earum.Lorem
-                    ipsum dolor sit amet consectetur adipisicing elit. Ipsa
-                    molestias at laudantium, distinctio nulla quo minima
-                    delectus consequuntur aut incidunt ipsam quidem repellat
-                    fugiat consequatur nihil odit quam voluptas earum.Lorem
-                    ipsum dolor sit amet consectetur adipisicing elit. Ipsa
-                    molestias at laudantium, distinctio nulla quo minima
-                    delectus consequuntur aut incidunt ipsam quidem repellat
-                    fugiat consequatur nihil odit quam voluptas earum.Lorem
-                    ipsum dolor sit amet consectetur adipisicing elit. Ipsa
-                    molestias at laudantium, distinctio nulla quo minima
-                    delectus consequuntur aut incidunt ipsam quidem repellat
-                    fugiat consequatur nihil odit quam voluptas earum.Lorem
-                    ipsum dolor sit amet consectetur adipisicing elit. Ipsa
-                    molestias at laudantium, distinctio nulla quo minima
-                    delectus consequuntur aut incidunt ipsam quidem repellat
-                    fugiat consequatur nihil odit quam voluptas earum.
-                  </p>
-                </Col>
-              </Row>
-            </form>
+        <Container className="mt-4 content-format">
+          <Row className="mx-auto flexForm">
+            <Row noGutters={true}>
+              <Col xs={12} sm={12} md={12} lg={7} className="mr-2">
+                <iframe
+                  src="https://docs.google.com/forms/d/e/1FAIpQLSe64AwGoRanKukVA4RzS-hzh_oN1EFMu4WQiWpKoCQ-LOO90w/viewform?embedded=true"
+                  width="100%"
+                  height="1000"
+                  frameborder="0"
+                  marginheight="0"
+                  marginwidth="0"
+                  onLoad={this.loaded}
```

Comment:
this we can also remove with the height css

Reply:
Thank you, Dan. I made all the changes for the dynamic iframe height response as per your comments here and also referencing your comments for feature/222/enroll page google form #258.